plugins/creative-writing-skills/skills/project-maintenance/resources/cli/cwcli/checks/prose_typography.py

```
@@ -0,0 +1,141 @@
+_LATIN_LETTER = re.compile(r"[A-Za-z]")
+
+_DIGIT_RUN_RE = re.compile(r"(?<!\d)\d{5,}(?!\d)")
+_DECIMAL_POINT_RE = re.compile(r"(?<=\d)\.(?=\d)")
+_NUMERO_RE = re.compile(r"\bNo\.\s*\d|#(?=\d)")
+# Malformed ordinals: any suffix glued to the digit without a hyphen, or a
+# full adjectival ending behind a hyphen. The minimal hyphenated forms
+# (1-й, 2-я, 3-е, 5-м, 5-х) and the two-letter conditioned forms (5-го,
+# 5-му, 20-ми) stay silent.
+_ORDINAL_RE = re.compile(
+    r"\d(?:ый|ой|ий|ая|ое|ые|ом|го|му|ми|й|я|е|м|х)\b"
+    r"|\d-(?:ый|ой|ий|ая|ое|ые|ом)\b"
```

Ruff (0.16.2)

[warning] String contains ambiguous `о` (CYRILLIC SMALL LETTER O). Did you mean `o` (LATIN SMALL LETTER O)?

(RUF001)